src/builtins/loaders/text-loader.js

```diff
@@ -0,0 +1,5 @@
+module.exports = function loadTextBundle(bundle) {
+  return fetch(bundle).then(function (res) {
+    return res.text();
```

Comment:
I would use an arrow function here 😀 

```javascript
module.exports.loadTextBundle = bundle => fetch(bundle).then(res => res.text())
```

Reply:
The linter (in the pre-commit hook) does not seems to like arrow functions:

```
src/builtins/loaders/text-loader.js
  1:26  error  Parsing error: Unexpected token >

✖ 1 problem (1 error, 0 warnings)


husky > pre-commit hook failed (add --no-verify to bypass)
```

Reply:
I was trying to commit this:

```js
module.exports = bundle => fetch(bundle).then(res => res.text())
```

Reply:
Oh ya builtins are supposed to be in ES5 👍